data/transport missions.txt

```diff
@@ -763,6 +772,24 @@ mission "There Might Be Riots part 2"
 			`	"It sounds like you were visiting Hai space," you say, and you describe the aliens who live beyond the wormhole. "Yeah, that would be them," says Ulrich. "I'd love to go back there some time, but for now we have commitments closer to home." You help them to unload their stuff, and say goodbye for now.`
 				accept
 
+mission "logbook: met TMBR before hai"
+	invisible
+	to offer
+		not "met hai before TMBR"
+		has "There Might Be Riots part 2: done"
+	on offer
+		log `Ulrich from There Might Be Riots talked about giant squirrel-like aliens. Maybe I can find where they are and bring the band there for an extended "drug vacation."`
+		fail
+		
+mission "logbook: met hai before TMBR"
+	invisible
+	to offer
+		has "met hai before TMBR"
+		has "There Might Be Riots part 2: done"
+	on offer
+		log `Told There Might Be Riots about the Hai. Maybe I can bring them to Hai space in the future, possibly for an extended "drug vacation."`
+		fail
+
```

Comment:
Both these need redone in some way (and if MZ doesn't want branching logs like this then one might need to go).